sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -187,6 +188,42 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates a notification when an Application Offering Change Request is completed by the ministry.
+   * @param notification input parameters to generate the notification.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param entityManager optional entity manager to execute in transaction.
+   */
+  async saveApplicationOfferingChangeRequestCompletedByMinistry(
+    notification: ApplicationOfferingChangeRequestCompletedByMinistryNotification,
```

Comment:
Minor: name seems to be very long and can be something like `ApplicationOfferingChangeRequestCompleteNotification`

Reply:
Every other method in the file is having the verb `save` associated. Is it fine if I keep it `saveApplicationOfferingChangeRequestCompleteNotification` ?